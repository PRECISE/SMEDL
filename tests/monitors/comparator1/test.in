nextString,
nextOpaque,
nextString,
nextOpaque,
nextString,a
nextOpaque,a
nextString,ab
nextOpaque,ab
nextString,a
nextOpaque,a
nextString,a
nextOpaque,a
nextString,
nextOpaque,
nextString,
nextOpaque,
nextString,abcd
nextOpaque,abcd
nextString,abce
nextOpaque,abce
nextString,bbce
nextOpaque,bbce
nextString,bbce
nextOpaque,bbce
