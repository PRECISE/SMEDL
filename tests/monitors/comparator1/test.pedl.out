_StringComp_match,
_OpaqueComp_match,
_StringComp_match,
_OpaqueComp_match,
_StringComp_mismatch,,a
_OpaqueComp_mismatch,,a
_StringComp_mismatch,a,ab
_OpaqueComp_mismatch,a,ab
_StringComp_mismatch,ab,a
_OpaqueComp_mismatch,ab,a
_StringComp_match,a
_OpaqueComp_match,a
_StringComp_mismatch,a,
_OpaqueComp_mismatch,a,
_StringComp_match,
_OpaqueComp_match,
_StringComp_mismatch,,abcd
_OpaqueComp_mismatch,,abcd
_StringComp_mismatch,abcd,abce
_OpaqueComp_mismatch,abcd,abce
_StringComp_mismatch,abce,bbce
_OpaqueComp_mismatch,abce,bbce
_StringComp_match,bbce
_OpaqueComp_match,bbce
